Classification/Core/src/main/java/org/tribuo/classification/baseline/DummyClassifierTrainer.java

```diff
@@ -39,7 +39,27 @@
  */
 public final class DummyClassifierTrainer implements Trainer<Label> {
```

Comment:
Not super happy with Dummy? Maybe BaselineClassifierTrainer would be better? Not specifically introduced by this PR, but maybe we should fix it while we're here?

Reply:
It mirrors scikit-learn's names for these kind of classifiers/regressors. We can change it in the next major version bump if you want.